flipper-native/android/src/debug/java/tech/bam/rnperformance/flipper/RNPerfMonitorPlugin.java

```diff
@@ -4,23 +4,46 @@
 import android.os.Looper;
 import android.util.Log;
 
+import androidx.annotation.NonNull;
+
+import com.facebook.flipper.android.AndroidFlipperClient;
 import com.facebook.flipper.core.FlipperConnection;
 import com.facebook.flipper.core.FlipperObject;
 import com.facebook.flipper.core.FlipperPlugin;
 import com.facebook.flipper.core.FlipperReceiver;
 import com.facebook.flipper.core.FlipperResponder;
 import com.facebook.react.ReactInstanceManager;
+import com.facebook.react.bridge.ReactApplicationContext;
 
 public class RNPerfMonitorPlugin implements FlipperPlugin {
     private FlipperConnection connection;
-    private ReactInstanceManager reactInstanceManager;
     private FPSMonitor fpsMonitor;
+    private ReactApplicationContext reactContext;
+
+    public static void passReactApplicationContext(@NonNull ReactApplicationContext reactContext) {
+		final RNPerfMonitorPlugin plugin = AndroidFlipperClient.getInstance(reactContext.getBaseContext())
+				.getPluginByClass(RNPerfMonitorPlugin.class);
 
+		if (plugin != null) {
+			plugin.setReactContext(reactContext);
+		} else {
+			Log.d(RNPerfMonitorPlugin.class.getName(), "Flipper plugin isn't installed");
+		}
+	}
+
+	// Keeping this constructor for retrocompatibility
+	@Deprecated
```

Comment:
```suggestion
```

Reply:
I'll remove the `@Deprecated` annotation for now in case we have a better fix from expo